keps/sig-storage/4049-storage-capacity-scoring-of-nodes-for-dynamic-provisioning/kep.yaml

```diff
@@ -0,0 +1,45 @@
+title: Storage Capacity Scoring of Nodes for Dynamic Provisioning
+kep-number: 4049
+authors:
+  - "@cupnes"
+owning-sig: sig-storage
+participating-sigs:
```

Comment:
Should sig-scheduling be a participating sig here?

Reply:
I addressed it.